manifest/base/projection-template.yaml

```diff
@@ -181,6 +183,8 @@ parameters:
   value: "1000"
 - name: ONPREM_EVENT_CHANNEL_BUFFER_SIZE
   value: "1000"
+- name: REDIS_PVC_NAME
```

Comment:
This change (and related) are here because it was changed the template directly, while we should change the kustomize templates first, and then generate the final templates :facepalm: